storage/innobase/xtrabackup/doc/source/glossary.rst

```diff
@@ -4,136 +4,199 @@
+   .exp
+     Files with the ``.exp`` extension are created by |Percona XtraBackup| per
+     each |InnoDB| tablespace when the :option:`--export` option is
+     used on prepare. See :doc:`restoring individual tables <xtrabackup_bin/restoring_individual_tables>`.
 
-.exp
-  Files with the ``.exp`` extension are created by *Percona XtraBackup* per each *InnoDB* tablespace when the `xtrabackup --export` option is used on prepare. These files can be used to import those tablespaces on *Percona Server for MySQL* 5.5 or lower versions, see :doc:`restoring individual tables <xtrabackup_bin/restoring_individual_tables>`".
+   .frm
+     For each table, the server will create a file with the ``.frm`` extension
+     containing the table definition (for all storage engines).
 
-.frm
-  For each table, the server creates a file with the ``.frm`` extension which contains the table definition (for all storage engines).
+   .ibd
+     On a multiple tablespace setup (:term:`innodb_file_per_table` enabled),
+     |MySQL| will store each newly created table on a file with a ``.ibd``
+     extension.
 
-.ibd
-  On a multiple tablespace setup (`innodb_file_per_table` enabled), *MySQL* will store each newly created table on a file with a ``.ibd`` extension.
+   .MRG
+     Each table using the :program:`MERGE` storage engine, besides of a
+     :term:`.frm` file, will have :term:`.MRG` file containing the names of the
+     |MyISAM| tables associated with it.
 
-.MYD
-  Each *MyISAM* table has ``.MYD`` (MYData) file which contains the data.
+   .MYD
+     Each |MyISAM| table has ``.MYD`` (MYData) file which contains the data on
+     it.
 
-.MYI
-  Each *MyISAM* table has ``.MYI`` (MYIndex) file which contains the table's indexes.
+   .MYI
+     Each |MyISAM| table has ``.MYI`` (MYIndex) file which contains the table's
+     indexes.
 
-.MRG
-  Each table using the `MERGE` storage engine, besides of an `.frm` file will have `.MRG` file containing the names of the *MyISAM* tables associated with it.
+   .opt
+     |MySQL| stores options of a database (like charset) in a file with a
+     :file:`.opt` extension in the database directory.
 
-.opt
-  *MySQL* stores options of a database (like charset) in a file with an `.opt` extension in the database directory.
+   .par
+     Each partitioned table has .par file which contains metadata about the
+     partitions.
 
-.par
-  Each partitioned table has .par file which contains metadata about the partitions.
+   .TRG
+     File containing the Triggers associated to a table, for example
+     `:file:`mytable.TRG`. With the :term:`.TRN` file, they represent all the
+     Trigger definitions.
 
-.TRG
-  File containing the Triggers associated to a table, for example,
-  ``mytable.TRG`. With the `.TRN` file, they represent all the
-  Trigger definitions.
+   .TRN
+     File containing the Triggers' Names associated to a table, for example
```

Comment:
Remove the extra space ___File
File containing the Triggers' Names associated with a table, for example,

Reply:
Rewritten.